org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/model/TypeHierarchyTests.java

```diff
@@ -3674,4 +3677,68 @@ public void testIndexQualificationBinaryNestedSubTypes_SearchForNestedSuperType(
 	}
 }
 
+/**
+ * @bug GitHub 269: Wrong type hierarchy computed for types with cyclic static imports.
+ * @see "https://github.com/eclipse-jdt/eclipse.jdt.core/issues/269"
+ */
+public void testBugGh269() throws Exception {
```

Comment:
Since we are now moved to GitHub, and everything is more cool and 50 times better, we have 2 bug trackers and will have 2 types if bug numbers. "Normal", old ones from the gone Eclipse bugzilla, and new ones from GitHub.

I don't know how to meaningfully name tests because of this. Should I ask if we have guidelines on cross platform dev, or do I assume nobody has even thought about this and just use whatever names I want?

Reply:
Please ask, I assume no one cared so far. "Gh269" makes sense for me.